Open Judge System/OJS.Common/Extensions/DirectoryHelpers.cs

```diff
@@ -57,8 +57,15 @@ public static void SafeDeleteDirectory(string path, bool recursive = false)
 
         public static void DeleteExecutionStrategyWorkingDirectories()
         {
-            var directoryPaths = Directory.GetDirectories(GlobalConstants.ExecutionStrategiesWorkingDirectoryPath);
-            foreach (var directoryPath in directoryPaths)
+            var executionStrategiesDirectoryPath = GlobalConstants.ExecutionStrategiesWorkingDirectoryPath;
```

Comment:
Directly use the constant, no need for an intermediary variable.

Reply:
It is actually a readonly field that calls `Path.Combine()` function, that's why I save it in variable first